src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -1093,6 +1093,38 @@ class @true {
                 SymbolDisplayPartKind.Punctuation);
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74117")]
+        public void TestBug74117()
```

Comment:
Consider using a descriptive name, perhaps `TestRecordStructName`.

Reply:
Makes sense. From the few places that I looked in Roslyn that had the `WorkItem` attribute, the ones I spotted went with the `TestBug123` convention, but looking further, I can see more that don't follow that convention.